phpunit.xml.dist

```diff
@@ -1,25 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit bootstrap="./tests/bootstrap.php" colors="true">
-    <testsuites>
-        <testsuite name="Assetic Test Suite">
-            <directory suffix="Test.php">./tests/Assetic/Test/</directory>
-        </testsuite>
-    </testsuites>
-
-    <php>
-        <server name="NODE_PATH" value="./node_modules/" />
-
-        <!-- <server name="CLOSURE_JAR" value="/path/to/google-closure/compiler.jar" /> -->
-        <!-- <server name="CSSEMBED_JAR" value="/path/to/cssembed.jar" /> -->
-        <!-- <server name="GSS_JAR" value="/path/to/closure-stylesheets.jar" /> -->
-        <!-- <server name="SPROCKETS_LIB" value="/path/to/sprockets/lib" /> -->
-        <!-- <server name="YUI_COMPRESSOR_JAR" value="/path/to/yuicompressor-2.4.2.jar" /> -->
-    </php>
-
-    <filter>
-        <whitelist>
-            <directory suffix=".php">./src/Assetic/</directory>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./tests/bootstrap.php" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage>
+    <include>
+      <directory suffix=".php">./src/Assetic/</directory>
+    </include>
+  </coverage>
+  <testsuites>
+    <testsuite name="Assetic Test Suite">
+      <directory suffix="Test.php">./tests/Assetic/Test/</directory>
+    </testsuite>
+  </testsuites>
+  <php>
+    <server name="NODE_PATH" value="./node_modules/"/>
+  </php>
```

Comment:
Please keep the original four spaces indentation

Reply:
Fixed, thanks for the hint!